dependencies.yaml

```diff
@@ -2,6 +2,8 @@ options:
   buildHeader: [ "load(\"@io_bazel_rules_scala//scala:scala_import.bzl\", \"scala_import\")" ]
   languages: [ "java", "scala:2.11.8" ]
   resolverType: "coursier"
+  strictVisibility: true
+  strictDeps: true
```

Comment:
Strict deps is not real is it?

Reply:
I misunderstood your last comment. I’ll remove both.